Source/WarpX.cpp

```diff
@@ -628,7 +633,7 @@ WarpX::AllocLevelData (int lev, const BoxArray& ba, const DistributionMapping& d
     int ngz_nonci = (ngz_tmp % 2) ? ngz_tmp+1 : ngz_tmp;  // Always even number
     int ngz;
     if (WarpX::use_fdtd_nci_corr) {
-        int ng = ngz_tmp + (mypc->nstencilz_fdtd_nci_corr-1);
+        int ng = ngz_tmp + 4;
```

Comment:
Instead of having a `4` here, maybe you could make this a static constexpr member of the NCIGodfreyFilter class? Then this would be a bit more self-describing:

```
int ng = ngz_tmp + NCIGodfreyFilter::stencil_width;
```